plugins/obs-webrtc/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+project(obs-webrtc)
+
+include(FetchContent)
+
+FetchContent_Declare(
+  Corrosion
+  GIT_REPOSITORY https://github.com/corrosion-rs/corrosion.git
+  GIT_TAG v0.3.1)
+
+FetchContent_MakeAvailable(Corrosion)
+
+set(obs-webrtc_HEADERS whip-output.h)
+set(obs-webrtc_SOURCES whip-service.c whip-output.c webrtc.c)
+
+if(WIN32)
```

Comment:
Use ‘if(OS_WINDOWS)` - support for the old OS-specific variables is soft-deprecated by CMake.

Reply:
Fixed d7f7e67